### DISCUSSION TITLE
Only displayed if necessary.....

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#94] [General]
This stuff could be hidden if not useful, in this case, only one line OR no data to display so pagination not needed to display on screen . I think....
![Screenshot 2023-05-26 at 20 25 38](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/9483e747-0c06-4e6c-a74f-7af4a524dad9)

**Comment by TJ-BitCanna** [comment]
Ah yes, that is a good point. Not showing more information than needed.

Let's see what we can do.

**Comment by TJ-BitCanna** [comment]
Suggested issue:
- Only show "Redelegates" and "Undelegates" blocks when they are applicable. Hide otherwise. If only one is active, make it pagewide.

**Comment by TJ-BitCanna** [comment]
These boxes are not included in the design, making this issue irrelevant.
